__tests__/app.test.js

```diff
@@ -60,43 +60,37 @@ describe('GET /api', () => {
     });
   });
 
-  describe("GET /api/articles/:article_id", () => {
-    test("200: should get an article by its id", () => {
-      return request(app)
-        .get("/api/articles/5")
+  describe('GET /api/articles', ()=>{
+    test('200: responds withan array of all article object excluding the body property', ()=>{
+        return request(app)
+        .get('/api/articles')
         .expect(200)
         .then(({body})=>{
-
-          const {article} = body
-            expect(article).toHaveProperty("article_id", 5);
+          const {articles} = body
+          expect(articles).toHaveLength(13)
+          articles.forEach((article)=>{
+            expect(article).toHaveProperty("article_id", expect.any(Number));
             expect(article).toHaveProperty("title", expect.any(String));
             expect(article).toHaveProperty("topic", expect.any(String));
             expect(article).toHaveProperty("author", expect.any(String));
-            expect(article).toHaveProperty("body", expect.any(String));
+            expect(article).not.toHaveProperty("body")
             expect(article).toHaveProperty("created_at", expect.any(String));
             expect(article).toHaveProperty("votes", expect.any(Number));
             expect(article).toHaveProperty("article_img_url", expect.any(String));
+          })
+            
         })
   
     });
-    test("should return error with msg Not Found for request not in database", () => {
-      return request(app)
-        .get("/api/articles/99999999")
-        .expect(404)
-        .then(({body})=>{
-          const {msg}= body
-          expect(msg).toBe("Not Found")
-        })
-  
-    });
-    test("should return error with msg Invalid Input for wrong user input", () => {
+    test('article should be ordered by dates in descending order',()=>{
       return request(app)
-        .get("/api/articles/hello")
-        .expect(400)
+        .get('/api/articles')
+        .expect(200)
         .then(({body})=>{
-          const {msg}= body
-          expect(msg).toBe("Bad request")
-        })
-  
-    });
-  });
+          const {articles} = body
+          expect(articles).toBeSorted('created_at')
+     
```

Comment:
As this is an array of objects, the docs for jest-sorted suggest that we should use `toBeSortedBy` as our matcher. You can check the docs here...
https://www.npmjs.com/package/jest-sorted